app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -74,6 +75,21 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       return window.location.search
     },
 
+    getSearchTerm: function () {
+      var queryString = this.getSearch()
```

Comment:
A bit unrelated (maybe more something to think about for tech debt) but I'm wondering if we should redact PII directly in the `getSearch` function?

Reply:
@andysellick That's probably a good idea :+1: